### PR TITLE
More magi

### PR DESCRIPTION
**Pull request by interfector**
New magi as I promised. 

I started to also add grogs from a post on Atlas Forum (as Medusa suggested) but it's a slow process. I also created a Non-Hermetic Grimoire and started to insert some spells and an Hermetic Grimoire to collect the various spells I already created. It may be useful to also create a Hermetic Magic Items and Non Hermetic Magic Items files to gather the various objects.

### CODE REVIEW

File: Grogs/Rhine Grogs.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
+# Rhine Grogs
+
```
Comment (GuillaumeDIDIER) [inline Grogs/Rhine Grogs.md:2]: It might be worth having a general intro of who are those grogs.

**Comment by GuillaumeDIDIER** [comment]
Sorry I'm being a bit busy right now.

File: Grogs/Rhine Grogs.md (hunk 1 above)
Comment (interfector) [reply Grogs/Rhine Grogs.md:2]: Mmm... Okay, I'll write a short intro referring to the Atlas post.

**Comment by interfector** [comment]
I've added a short description, I probably could've said that the grog list will be expanded.

**Pull request merged**